### PR TITLE
Fixed the demo for crosshair label.

### PR DESCRIPTION
**Pull request by kamil-musialowski**
Original demo didn't have the `yAxis crosshair label` enabled. 

Since the same demo is also used for showing the crosshair functionality, I decided to change the title instead of creating a new one (since the differences between label enabled/disabled are minor).

**Old:** https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/xaxis/crosshairs-xy/
**New:** https://jsfiddle.net/BlackLabel/rj83w0h5/

**API:** https://api.highcharts.com/highstock/yAxis.crosshair
https://api.highcharts.com/highstock/yAxis.crosshair.label.enabled

### CODE REVIEW

File: ts/Core/Axis/AxisDefaults.ts [hunk 1]
```diff
@@ -232,7 +232,7 @@ namespace AxisDefaults {
          * @sample {highcharts} highcharts/xaxis/crosshair-both/
          *         Crosshair on both axes
          * @sample {highstock} stock/xaxis/crosshairs-xy/
-         *         Crosshair on both axes
+         *         Crosshair on both axes, with yAxis label
```
Comment (TorsteinHonsi) [inline ts/Core/Axis/AxisDefaults.ts:235]: Code must be either be written in backticks, `yAxis` or in plain English: y axis. I think plain English works best in this case.

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks! A minor proofreading note.

File: ts/Core/Axis/AxisDefaults.ts (hunk 1 above)
Comment (kamil-musialowski) [reply ts/Core/Axis/AxisDefaults.ts:235]: Corrected, thank you!

**Review by TorsteinHonsi — APPROVED** [review]
